### ISSUE TITLE
[CAIA Intake] Benefits Non-Disability Experience Team: VR&E Confirmation Email Copy Review

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#74736]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Benefits Non-Disability Experience 
- **OCTO product owner:** Emily Theis
- **Product name:** Veteran Readiness and Employment (VR&E) aka Chapter 31 (28-1900)
- **Product manager:** Laura Steele
- **Slack channel:** #benefits-non-disability
- **Dedicated content writer on your team (if you have one):** none
- **Dedicated a11y specialist on your team (if you have one):** none
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [x] Something else: Update the confirmation email Veterans receive after submitting on online VR&E (28-1900) application.

### What's the nature of your initiative and desired outcomes?

The VBA reached out to our team requesting that the decision timeline mentioned in the existing VR&E confirmation email be updated because they can no longer meet the 1 week review timeline the email mentions. As part of this update, we would like CAIA to review the email copy for any additional updates/improvements that may be needed. Our goal is to ensure Veterans receiving this confirmation email understand:
- The VA received their VR&E claim on xx date
- What will happen next
- When they can expect a decision

Note: We also discovered that the timeline will need to be updated on the form submission page (see screenshot in supporting artifacts).

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?

We have captured the [copy of the current confirmation email](https://docs.google.com/document/d/1Q7ns4wmVe8tO2N4CtUvj_li9jGMVJRq6vMskAs0rVZw/edit?usp=sharing), and we're working with VBA SMEs to understand what the decision timeline should be changed to (right now the email states 1 week). We anticipate that the update to the decision timeline language will be minor (e.g., change 1 week to 4 weeks), so we wanted to kick off a CAIA review of the email copy while we wait for the final timeline language from the VBA. We would like to have the email updated by the end of February 2024.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [x] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `xxxxxx`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Current Email Copy and Example Template](https://docs.google.com/document/d/1Q7ns4wmVe8tO2N4CtUvj_li9jGMVJRq6vMskAs0rVZw/edit?usp=sharing) (needs CAIA review)
- [How to Apply for VR&E](https://www.va.gov/careers-employment/vocational-rehabilitation/how-to-apply/)
- [Confirmation page](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1684161597392/5c1f8869cf5d03b792c8b82cf19b39dc26c486c2?wid=0-1705005793703) (mentions decision timeline): 
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/134541644/98c3912b-3e4c-437f-ae57-223ed6f2960f)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [x] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [x] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA content tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/75608
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76947
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76949
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/76949
```

**Comment by steele-lm** [comment]
@fiorella-io @jstrothman

**Comment by laurwill** [comment]
@steele-lm , could you provide more information about the e-VA system mentioned in the email? We weren't able to find a current webpage or entry point for this tool. Thanks!

**Comment by steele-lm** [comment]
@steele-lm We don't know a lot about it, but it's a [system that was developed for the VR&E program](https://news.va.gov/73853/vre-pilots-electronic-virtual-assistant-e-va/), and it facilitates scheduling appointments. This is how it is currently described within the form, but we're removing this page entirely at the direction of the VBA because Veterans automatically receive an email asking them to opt into e-VA after their VR&E application is processed. We opted to add language about it in the email instead, given that it's a follow-on activity to submitting the form:
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/134541644/c3d413dd-0c11-4bb9-baa4-7dd2927a7f26)

**Comment by fiorella-io** [comment]
@laurwill This is what I found online:
- [VA news article](https://news.va.gov/press-room/va-offers-electronic-virtual-assistant-for-veterans-to-connect-faster-with-counselors/#:~:text=The%20electronic%20Virtual%20Assistant%20(e,to%20schedule%20and%20reschedule%20appointments.)

**Comment by laurwill** [comment]
Thanks @fiorella-io ! The [VBA page](https://www.benefits.va.gov/vocrehab/) has a promo about e-VA, but it links to our VR&E page on VA.gov, which doesn't mention this service. It sounds from Laura's post above that VBA stakeholders confirmed that the e-VA system is still in use. Is that right?

**Comment by steele-lm** [comment]
@laurwill Yes, the e-VA system is still in-use, so I do think we have an opportunity to call it out on the VR&E pages on VA.gov.

**Comment by katherine-fung** [comment]
Hi @steele-lm @fiorella-io, 

I want to confirm some content in the current email. Does the information in bold apply only to people who are **approved** for VR&E? 

After you apply, we’ll schedule a meeting for you with a Vocational Rehabilitation Counselor (VRC) to find out if you’re eligible for VR&E benefits and services. **You will have the option to opt into using our private and secure Electronic-Virtual Assistant (e-VA), which allows you to schedule appointments, contact your counselor, and send documentation online.After we make a decision, you and your counselor will work together to develop a rehabilitation plan, which outlines the services available to you to become successful.**

cc @aprocik1 @laurwill

**Comment by katherine-fung** [comment]
Hi @steele-lm @fiorella-io, 

I've provided the content recommendations for the email in the Google doc: 

https://docs.google.com/document/d/1Q7ns4wmVe8tO2N4CtUvj_li9jGMVJRq6vMskAs0rVZw/edit 

Let me know if you have questions or feedback. If you have the VA Notify link, I can also draft this copy there so you can see what it looks like in email format.

**Comment by fiorella-io** [comment]
Thank you, @katherine-fung. I just shared this updated email draft with our SMEs over Teams. Once we hear back from them we can go ahead and add it to VA Notify. I need to check if I still have access. If I do, I'll share the VA Notify link.

**Comment by katherine-fung** [comment]
Hi @fiorella-io @steele-lm, 

The SMEs have approved the latest plain language edits to the email copy. The copy is in the source of truth doc: 

https://docs.google.com/document/d/1Q7ns4wmVe8tO2N4CtUvj_li9jGMVJRq6vMskAs0rVZw/edit

Thanks!

**Comment by fiorella-io** [comment]
Thank you so much @katherine-fung for working with our SMEs and finalizing the copy! I'll add the copy to notify and notify our SMEs once it has been pushed to production. Thank you!
